package.json

```diff
@@ -33,7 +33,7 @@
     "build-scss": "scss-bundle",
     "extract-scss-variables": "extract-scss-variables ./dist/archwizard.scss ./dist/variables.scss",
     "build-css": "node-sass ./dist/archwizard.scss ./dist/archwizard.css",
-    "test": "ng test",
+    "test": "./testIvy.sh false && ng test && ./testIvy.sh true && ng test || (./testIvy.sh true && exit 1)",
```

Comment:
Ideally I would like to get it working without adding a shellscript to run the tests both with Ivy enabled and disabled.
It seems like `ng test` supports a `--tsConfig` parameter (see also [here](https://angular.io/cli/test)). Maybe we could add an additional `tsconfig` file with

```json
  "angularCompilerOptions": {
    "enableIvy": true
  }
```

instead, which we can then load using the `--tsConfig` parameter?

Reply:
oh didn't know about it. Gonna test.